Services/Services/UserService.cs

```diff
@@ -1,92 +1,110 @@
 using Common.Listing;
 using Data.Entities;
 using Data.Repositories;
-using Microsoft.AspNetCore.Identity;
-using PagedList;
-using Services.DTOs;
 using PagedList;
 using Services.DTOs.User;
 
 namespace Services.Services
 {
     public class UserService
     {
-        private UserRepository _userRepository;
-
-        public UserService(UserRepository userRepository) { _userRepository = userRepository; }
-
-        public async Task<int> AddUser(UserDTO dto)
+        private UserRepository userRepository;
 
+        public UserService(UserRepository _userRepository)
         {
-            return 0;
+            userRepository = _userRepository;
         }
-        public void UpdateUser(int id, UserDTO dto)
+
+        public UserDTO Get(int? userId)
         {
+            User user = userRepository.GetUserById(userId);
+            if (user == null)
```

Comment:
If user is null you are returnig null. Be if you returned user it would be also null. Just return user